src/FloatParser.php

```diff
@@ -31,4 +31,14 @@ public function required($invalidValueMessage = null, $notFoundMessage = null)
     {
         return parent::required($invalidValueMessage, $notFoundMessage);
     }
+
+    /**
+     * @param float $minValue
+     * @param float $maxValue
+     * @return FloatInRangeParser
+     */
+    public function inRange($minValue, $maxValue)
```

Comment:
Maybe we could call this `between` (the idea just popped up in my head and it feels like a better name than `inRange`, kind of more natural to say "between" than to say "in range")? :)